config/v1/types_network.go

```diff
@@ -13,20 +13,50 @@ type Network struct {
 	// Standard object's metadata.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// spec holds user settable values for configuration
+	// Spec holds user settable values for configuration.
```

Comment:
@deads2k mentioned in https://github.com/openshift/api/pull/139#discussion_r236243011
`spec` to match the json tag